src/imex.rs

```diff
@@ -106,6 +107,178 @@ pub async fn imex(
     res
 }
 
+pub async fn receive_backup(
```

Comment:
would be nice to have a doc comment here as well, including describing its interactions with the "ongoing" system.